Resources/public/templates/rawcontent.hbt

```diff
@@ -1,4 +1,5 @@
 <h2 class="ez-raw-content-title"><a href="#">Content</a></h2>
+<div class="ez-language-switcher"></div>
```

Comment:
usually we add the `-container` on such element so it should be called `ez-language-switcher-container`.